packages/wonder-blocks-data/src/index.js

```diff
@@ -52,6 +52,7 @@ export {useServerEffect} from "./hooks/use-server-effect.js";
 export {useRequestInterception} from "./hooks/use-request-interception.js";
 export {useSharedCache, clearSharedCache} from "./hooks/use-shared-cache.js";
 export {ScopedInMemoryCache} from "./util/scoped-in-memory-cache.js";
+export {resultFromCachedResponse} from "./util/result-from-cache-response.js";
```

Comment:
The `useServerEffect` hook returns the `CachedResponse` type. I think we want to keep it that way rather than have it return the `Result` type, though I could be convinced otherwise. So, rather than change that hook, I export the utility that can convert from the `CachedResponse` type to the `Result` type.

We could, in future remove this and just keep `CachedResponse` as an internal type instead, but for now, this will enable us to do the things we need to do without code duplication :)